src/commands/init.ts

```diff
@@ -80,59 +66,26 @@ export class InitCommand implements Command {
           } else {
             logger.warn(`Template ${options.name} not found`);
             logger.debug(`Yeoman generator ${generatorName} not found`);
+            reject(`Template ${options.name} not found`);
           }
         };
 
-        let getDisplayName = (generatorName) => {
-          let nameEnd = generatorName.indexOf(':');
-          if (nameEnd === -1) nameEnd = generatorName.length;
-          return generatorName.substring('polymer-init-'.length, nameEnd);
-        };
-
         if (options.name) {
           let generatorName = `polymer-init-${options.name}:app`;
           runGenerator(generatorName, options.name);
         } else {
           let polymerInitGenerators = Object.keys(generators)
-              .filter((k) => k.startsWith('polymer-init')
-                  && k !== 'polymer-init:app');
+            .filter((k) => k.startsWith('polymer-init')
```

Comment:
Did tslint force the indentation removal. If so, ok. If not, I prefer indenting again for expressions that are line broken.